src/ctr_exit.c

```diff
@@ -82,8 +82,7 @@ gboolean on_signalfd_cb(gint fd, G_GNUC_UNUSED GIOCondition condition, gpointer
 	struct pid_check_data *data = (struct pid_check_data *)user_data;
 
 	/* dequeue the signal from the signalfd */
-	int sig = dequeue_signal_event(fd);
-	g_assert_cmpint(sig, ==, SIGCHLD);
+	dequeue_signal_event(fd);
```

Comment:
Isn't this going to choke at compile time because of missing `(void)`?

Reply:
I've changed the function to be `void`